config/src/parameters/user.rs

```diff
@@ -236,10 +224,9 @@ impl Root {
 }
 
 #[derive(Copy, Clone)]
-pub struct CliContext {
-    pub submit_genesis: bool,
-}
+pub struct CliContext {}
```

Comment:
If there is no context needed from CLI layer for configuration parsing anymore, I think this struct may be removed completely.